pkg/forklift-api/webhooks/mutating-webhook/mutators/secret-mutator.go

```diff
@@ -38,14 +40,17 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 	}
 
 	var insecure = false
-	if insecureSkipVerify, ok := secret.Data["insecureSkipVerify"]; ok {
-		insecure, err = strconv.ParseBool(string(insecureSkipVerify))
-		if err != nil {
-			log.Error(err, "Failed to parse insecure property from the secret")
-			return util.ToAdmissionResponseError(err)
+	if _, ok := secret.GetLabels()["createdForProviderType"]; ok {
```

Comment:
this check is redundant - we forward 'create' and 'update' operations to this webhook with objectSelector that checks the `createdForProviderType` exists so we can just replace this with a comment to make it clear that this is the assumption here